index.js

```diff
@@ -1,39 +1,29 @@
-module.exports = robot => {
-  robot.on('issues', async context => {
-    if (!context.payload.repository.owner.login || !context.payload.issue.user.login || !context.payload.repository.name) {
-        issue = (await context.github.issues.get(context.issue())).data;
-    }
-
-    const user_login = context.payload.issue.user.login;
-    const repo_owner_id = context.payload.repository.owner.login;
-    const repo_name = context.payload.repository.name;
+const checkCount = require('./lib/checkCount');
 
-    const options = context.repo({path: '.github/new-issue-welcome.md'});
-    const res = await context.github.repos.getContent(options);
-    const template = new Buffer(res.data.content, 'base64').toString();
+module.exports = robot => {
+    robot.on('issues.opened', async context => {
+        const userLogin = context.payload.issue.user.login;
+        const repoOwnerLogin = context.payload.repository.owner.login;
+        const repoName = context.payload.repository.name;
 
-    const github = await robot.auth(context.payload.installation.id);
-    const response = await github.issues.getForRepo(context.repo({
-        owner: repo_owner_id,
-        repo: repo_name,
-        state: "all",
-        creator: user_login
-    }));
+        const github = await robot.auth(context.payload.installation.id);
+        const response = await github.issues.getForRepo(context.repo({
+            owner: repoOwnerLogin,
+            repo: repoName,
+            state: 'all',
+            creator: userLogin
+        }));
 
-    var count_issues = 0
-    //check for issues that are also PRs
-    for (i = 0; i < response.data.length; i++) {
-        if (!((response.data[i]).pull_request)) {
-            count_issues += 1
-        }
-        //exit loop if more than one PR
-        if (count_issues > 1) {
-            break;
+        if (checkCount.PRCount(response)) {
+            let template;
+            try {
+                const options = context.repo({path: '.github/new-issue-welcome.md'});
+                const res = await context.github.repos.getContent(options);
+                template = Buffer.from(res.data.content, 'base64').toString();
+            } catch (err) {
```

Comment:
I'm not sure. I looked here https://developer.github.com/v3/repos/contents/ and didn't see anything that accomplished that? Should I just check that a response exists and go from there?